DMX.Core.Api.Tests.Unit/Services/Orchestrations/LabOrchestrationServiceTests.Logic.RetrieveById.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.Foundations.Labs;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Orchestrations
+{
+    public partial class LabOrchestrationServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveLabFromStorageByIdAsync()
+        {
+            // given
+            Guid randomLabId = Guid.NewGuid();
+            Guid inputLabId = randomLabId;
+            Lab randomLab = CreateRandomLab();
+            Lab selectedLab = randomLab;
+            Lab expectedLab = selectedLab.DeepClone();
+
+            this.labServiceMock.Setup(service =>
+                service.RetrieveLabByIdAsync(inputLabId))
+                    .ReturnsAsync(selectedLab);
```

Comment:
Since the method here says `RetrieveLabByIdAsync` then our variable should match that. so put there `retrievedLab` instead of `selectedLab`